assets/js/signup.js

```diff
@@ -79,6 +68,40 @@ var loadQuestions = function() {
       validationObject["rules"] = rules;
       validationObject["messages"] = messages;
       validateForm(validationObject);
+
+      //Since non-required fields are "valid" when they are empty, we need an
+      //alternate way to keep labels raised when there is content in their
+      //associated input field
+      $( ".vita-form-textfield input" ).focusout(function() {
+        if($.trim($(this).val()).length > 0) {
```

Comment:
Consider:
```js
var isBlank = $.trim($(this).val()).length > 0;
$label = $(this).closest(".vita-form-textfield").find(".vita-form-label").toggleClass( "vita-form-label__floating", isBlank );
```
jQuery's `toggleClass()` has a second parameter option. Documentation can be found [here](http://api.jquery.com/toggleclass/).